list1.py

```diff
@@ -37,8 +41,18 @@ def match_ends(words):
 # Hint: this can be done by making 2 lists and sorting each of them
 # before combining them.
 def front_x(words):
```

Comment:
We can cut out a lot of code here, just like you did on the sort_last function:

x_words = [x for x in words if x[0] == x]
words = same thing but;  != x
return x_words.sort() + words.sort()